Automattic/jetpack
# External Media: Prevent clicks from propagating out

Stops click propagation where they're occurring within the External Media modal and its buttons.

Fixes #16070.

#### Changes proposed in this Pull Request:

* Prevents clicks from propagating past the External Media button.


#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Create a Gallery block with a few images.
* In the Upload bar at the bottom of the block, click on "Select images"
* Select and insert images.
* Check if the upload modal appears, as described in #16070.

#### Proposed changelog entry for your changes:

* None needed.
